src/main/java/de/mediathekview/mserver/crawler/orf/OrfTopicUrlDTO.java

```diff
@@ -0,0 +1,34 @@
+package de.mediathekview.mserver.crawler.orf;
+
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import java.util.Objects;
+
+public class OrfTopicUrlDTO extends CrawlerUrlDTO {
+  
+  private final String theme;
+  
+  public OrfTopicUrlDTO(String aTheme, String aUrl) {
+    super(aUrl);
+    theme = aTheme;
+  }
+  
+  public String getTheme() {
+    return theme;
+  }
+  
+  @Override
+  public boolean equals(final Object obj) {
+    if (super.equals(obj)) {
+      return getTheme().equals(((OrfTopicUrlDTO)obj).getTheme());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "obj" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)